data/saas/config/gorgias_config.yml

```diff
@@ -0,0 +1,134 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Gorgias SaaS Config
+  type: gorgias
+  description: A sample schema representing the Gorgias connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: ethyca-test.gorgias.com
```

Comment:
This domain is customer-specific so we can't use it as the default value

Reply:
default domain value removed